pkg/icingadb/ha.go

```diff
@@ -274,15 +287,25 @@ func (h *HA) realize(ctx context.Context, s *icingaredisv1.IcingaStatus, t *type
 			switch errQuery {
 			case nil:
 				otherResponsible = true
-				if shouldLog {
+				if shouldLogRoutineEvents {
 					h.logger.Infow("Another instance is active",
 						zap.String("instance_id", instance.Id.String()),
 						zap.String("environment", envId.String()),
-						"heartbeat", instance.Heartbeat,
+						zap.Time("heartbeat", instance.Heartbeat.Time()),
 						zap.Duration("heartbeat_age", time.Since(instance.Heartbeat.Time())))
 				}
 			case sql.ErrNoRows:
-				takeover = true
+				fields := []any{
+					zap.String("instance_id", h.instanceId.String()),
+					zap.String("environment", envId.String()),
+					zap.Duration("heartbeat_timeout", timeout)}
```

Comment:
As an administrator using Icinga DB, I might wanna know what timeout was exceeded as this might be a helpful debug information. Thus, as I just wrote in <https://github.com/Icinga/icingadb/pull/692#discussion_r1530070562>, this would only be `DEBUG`-logged.

Reply:
On second thoughts, this is not really helpful after all.